tests/forms_tests/field_tests/test_decimalfield.py

```diff
@@ -195,31 +194,22 @@ def test_decimalfield_changed(self):
             localized_d = formats.localize_input(d)  # -> '0,1' in French
             self.assertFalse(f.has_changed(d, localized_d))
 
-    # RemovedInDjango50Warning: When the deprecation ends, remove
-    # @ignore_warnings and USE_L10N=False. The test should remain because
-    # format-related settings will take precedence over locale-dictated
-    # formats.
```

Comment:
Locale-dictated formats always takes precedence over format-related settings, so I made a mistake preparing these comments :disappointed:

Reply:
@claudep Removing `USE_L10N` was really tricky :tired_face:  I would be grateful for a review :gift: